in_toto/runlib.go

```diff
@@ -96,6 +96,15 @@ func RecordArtifacts(paths []string, hashAlgorithms []string, gitignorePatterns
 	// Make sure to initialize a fresh hashset for every RecordArtifacts call
 	visitedSymlinks = NewSet()
 	evalArtifacts, err = recordArtifacts(paths, hashAlgorithms, gitignorePatterns, lStripPaths, lineNormalization, followSymlinkDirs)
+	// normalize windows paths to unix paths
+	for key, value := range evalArtifacts {
+		delete(evalArtifacts, key)
```

Comment:
Modifying a map during iteration can sometimes lead to unpredictable behavior (though probably more when we're adding new keys). I recommend updating this to `evalArtifactsUnnormalized` or something and then creating a new `evalArtifacts` with the path normalized for each element in the former.